tripal/tripal_jbrowse
# Register Existing JBrowse instances with Tripal JBrowse Management

Issue #25 

This PR adds the ability to register existing JBrowse instances with Tripal JBrowse Management. Specifically, it adds an "Register Existing Instance" action link on the instances listing page, which will create the Tripal JBrowse management record but not initialize the data directory, etc.

## Use Cases
A) Provides an upgrade path for users of the old Tripal JBrowse Page Integration module.
B) Provides a way to re-register an instance you accidentally deleted (assuming you did not run the tripal job).

## Testing
1. Create an instance using the "Create New Instance" and ensure it still works correctly.
2. Delete the instance and cancel the Tripal job created.
3. Re-register the instance you just deleted and check you can add new tracks.

NOTE: un-managed tracks will now be shown on the Instance page. These are read from the JSON but will not have an "manage" or "delete" link as this functionality is non-trivial.
